aws-ts-ec2-provisioners/index.ts

```diff
@@ -54,11 +55,12 @@ const server = new aws.ec2.Instance("server", {
     keyName: keyName,
     securityGroups: [ secgrp.name ],
 });
-const conn = {
+const conn: provisioners.ConnectionArgs = {
     host: server.publicIp,
     username: "ec2-user",
     privateKey,
-    privateKeyPassphrase,
+	privateKeyPassphrase,
+	changeToken: getFileHash("myapp.conf"),
```

Comment:
Some indentation appears to be off here?

Reply:
Yeah my editor was set to use Tabs for this repo for some reason. I have fixed it now.